build-tools/packages/build-cli/package.json

```diff
@@ -143,7 +143,7 @@
 		"typescript": "~4.5.5"
 	},
 	"engines": {
-		"node": ">=14.0.0"
```

Comment:
This is also a breaking change. Thats probably fine to require the same version of node for running our build tools as we do for development.

Reply:
Same applies to the other changes in buil-tools